rust_src/src/marker.rs

```diff
@@ -60,6 +62,14 @@ impl LispMarkerRef {
     }
 }
 
+impl Iterator for LispMarkerRef {
+    type Item = LispMarkerRef;
+
+    fn next(&mut self) -> Option<LispMarkerRef> {
+        unsafe { self.next.as_ref().map(|o| mem::transmute(o)) }
```

Comment:
Is this a variation on `as_marker()` we can capture? It appears a couple of times in this PR.